alacritty/src/input.rs

```diff
@@ -868,6 +868,30 @@ impl<'a, T: EventListener, A: ActionContext<T>> Processor<'a, T, A> {
         self.ctx.window_mut().set_mouse_cursor(mouse_state.into());
     }
 
+    // Same logic to map to control characters as found in
+    // libX11's _XTranslateKeySym function.
```

Comment:
I'm still trying to understand all this, but in the process found this useful message on the xorg mailing list: https://www.mail-archive.com/xorg@lists.x.org/msg04434.html

Reply:
I mean, we currently do all those changes internally inside libX11, so we should probably keep things the same, just to not have a difference between both keys.